uniflow/op/model/cv/model_server.py

```diff
@@ -262,3 +268,214 @@ def __call__(self, data: List[str]) -> List[str]:
             out = re.sub(r"\n{3,}", "\n\n", final_md.strip()).strip()
             outs.append(out)
         return outs
+
+
+class LlamaParseModelServer(AbsModelServer):
+    """Llama Parse Model Server"""
+
+    def __init__(
+        self, prompt_template: PromptTemplate, model_config: Dict[str, Any]
+    ) -> None:
+        super().__init__(prompt_template, model_config)
+        self._model_config = LlamaParseModelConfig(**self._model_config)
+
+        try:
+            import nest_asyncio  # pylint: disable=import-outside-toplevel
+
+        except ModuleNotFoundError as exc:
+            raise ModuleNotFoundError(
+                "Please install llama_parse to use LlamaModelServer. You can use `pip install llama_parse` to install it."
+            ) from exc
+
+    def _preprocess(self, data: str) -> List[str]:
+        """Preprocess data.
+
+        Args:
+            data (List[str]): Data to preprocess.
+
+        Returns:
+            List[str]: Preprocessed data.
+        """
+        return data
+
+    def _postprocess(self, data: List[str]) -> List[str]:
+        """Postprocess data.
+
+        Args:
+            data (List[str]): Data to postprocess.
+
+        Returns:
+            List[str]: Postprocessed data.
+        """
+        return data
+
+    def __call__(self, data: List[str]) -> List[str]:
+        """Run model.
+
+        Args:
+            data (List[str]): Data to run.
+
+        Returns:
+            List[str]: Output data.
+        """
+
+        import os
+
+        import nest_asyncio
+
+        nest_asyncio.apply()
+
+        from llama_parse import LlamaParse
+
+        parser = LlamaParse(
+            api_key=os.getenv("LLAMA_CLOUD_API_KEY"),
+            result_type=self._model_config.result_type,  # "markdown" and "text" are available
+            num_workers=self._model_config.num_wokers,  # if multiple files passed, split in `num_workers` API calls
+            verbose=True,
+            language=self._model_config.language,  # Optionally you can define a language, default=en
+        )
+
+        out = []
+
+        if self._model_config.sync:
+            for pdf in data:
+                result = parser.load_data(pdf)
+                out.append(result[0].text)
+        else:
+            # TODO: implement async
+            print("running llamaparse async...")
```

Comment:
nit: we should raise `notimplementederror` instead of handling silent here without doing anything.